packages/components/socialFeed/RichText/RichText.tsx

```diff
@@ -34,28 +35,35 @@ export const RichText: React.FC<RichTextProps> = ({
   const [initialHeight, setInitialHeight] = useState(windowHeight);
 
   const handleHeightChange = useCallback((height: number) => {
-    setInitialHeight(height * 2);
```

Comment:
@omniwired  I am not sure why ```height * 2``` was used. This makes lots of spacing after the article and before the comment. So, I removed it.